website/docs/d/client_config.html.markdown

```diff
@@ -30,8 +30,7 @@ There are no arguments available for this data source.
 * `client_id` is set to the Azure Client ID (Application Object ID).
 * `tenant_id` is set to the Azure Tenant ID.
 * `subscription_id` is set to the Azure Subscription ID.
-* `service_principal_application_id` is the Service Principal Application ID.
-* `service_principal_object_id` is the Service Principal Object ID.
+* `object_id` is set to the Azure Object ID.
```

Comment:
we should also call out this will be empty when using ADFS?

```suggestion
* `object_id` is set to the Azure Object ID.

-> **Note:** This is only applicable when not using ADFS.
```